data_model/clusters/DoorLock.xml

```diff
@@ -0,0 +1,2296 @@
+
+1 - The ability to modify Users/Credentials/Schedules locally is enabled">
+          <mandatoryConform/>
+        </bitfield>
+        <bitfield name="Clear Locally" bit="2" summary="0 - The ability to clear Users/credentials/Schedules locally is disabled
+
+1 - The ability to clear Users/Credentials/Schedules locally is enabled">
+          <mandatoryConform/>
+        </bitfield>
+        <bitfield name="Adjust Locally" bit="3" summary="0 - The ability to adjust lock settings locally is disabled
+
+1 - The ability to adjust lock settings locally is enabled">
+          <mandatoryConform/>
+        </bitfield>
+      </bitmap>
+      <access read="true" write="optional" readPrivilege="view" writePrivilege="admin"/>
+      <quality changeOmitted="false" nullable="false" scene="false" persistence="volatile" reportable="true"/>
+      <optionalConform/>
+    </attribute>
+    <attribute id="0x0030" name="WrongCodeEntryLimit" type="uint8" default="MS">
+      <access read="true" write="optional" readPrivilege="view" writePrivilege="admin"/>
+      <quality changeOmitted="false" nullable="false" scene="false" persistence="volatile" reportable="true"/>
+      <mandatoryConform>
+        <orTerm>
+          <feature name="PIN"/>
+          <feature name="RID"/>
+        </orTerm>
+      </mandatoryConform>
+      <constraint type="between" from="1" to="255"/>
+    </attribute>
+    <attribute id="0x0031" name="UserCodeTemporaryDisableTime" type="uint8" default="MS">
+      <access read="true" write="optional" readPrivilege="view" writePrivilege="admin"/>
+      <quality changeOmitted="false" nullable="false" scene="false" persistence="volatile" reportable="true"/>
+      <mandatoryConform>
+        <orTerm>
+          <feature name="PIN"/>
+          <feature name="RID"/>
+        </orTerm>
+      </mandatoryConform>
+      <constraint type="between" from="1" to="255"/>
+    </attribute>
+    <attribute id="0x0032" name="SendPINOverTheAir" type="bool" default="0">
+      <access read="true" write="optional" readPrivilege="view" writePrivilege="admin"/>
+      <quality changeOmitted="false" nullable="false" scene="false" persistence="volatile" reportable="true"/>
+      <optionalConform>
+        <andTerm>
+          <notTerm>
+            <feature name="USR"/>
+          </notTerm>
+          <feature name="PIN"/>
+        </andTerm>
+      </optionalConform>
+    </attribute>
+    <attribute id="0x0033" name="RequirePINforRemoteOperation" type="bool" default="0">
+      <access read="true" write="optional" readPrivilege="view" writePrivilege="admin"/>
+      <quality changeOmitted="false" nullable="false" scene="false" persistence="volatile" reportable="true"/>
+      <mandatoryConform>
+        <andTerm>
+          <feature name="COTA"/>
+          <feature name="PIN"/>
+        </andTerm>
+      </mandatoryConform>
+    </attribute>
+    <attribute id="0x0034" name="SecurityLevel" default="0">
```

Comment:
This attribute has no data type

Reply:
I guess we could skip it given deprecateConform?

Reply:
Yeah, there's actually a fair number of warnings emitted when the XMLs get generated, all of which should be backported either to a scraper issue or a spec issue. That's kind of orthogonal to this PR though.

I opened a spec issue to track - I don't have time to go through these right now, but I suspect some of these are just not correct in the spec. https://github.com/CHIP-Specifications/connectedhomeip-spec/issues/7680

Reply:
Going to track the spec weirdness in that issue, and close off this comment to get the PR in. We can update the XMLs as we pull in the spec fixes.